tools/site-creator/site-creator.html

```diff
@@ -22,7 +22,7 @@
     <h1>Initialize your storefront content</h1>
     <img width="2000" height="558" class="logo" src="./Logo-Adobe-Commerce.png" />
   </div>
-  <p class="subheader">This tool will clone the content behind <a href="https://www.aemshop.net" target="_blank">aemshop.net</a> to your own content directory. After you create the github repo, please make sure you onboard your site to <a href="https://www.aem.live/docs/config-service-setup" target="_blank">AEM config service</a>, or commit a <a href="https://main--aem-boilerplate-commerce--hlxsites.aem.live/config.json" target="_blank">config.json</a> with your Commerce environment details to your code repo.</p>
+  <p class="subheader">This tool will clone the content behind <a href="https://www.aemshop.net" target="_blank">aemshop.net</a> to your own content directory. After you create the GitHub repository, please make sure that you install the <a href="https://github.com/apps/aem-code-sync" target="_blank">AEM Code Sync bot</a> and that you onboard your site to <a href="https://www.aem.live/docs/config-service-setup" target="_blank">AEM config service</a>, or commit a <a href="https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/default-fstab.yaml" target="_blank">fstab.yaml</a> with your org and site name to your code repo.</p>
```

Comment:
I'm not sure why you have this as changed in your PR - it is already on main... I guess this is fine?

Reply:
I don't think `integration` has the latest from `main`. It should go away once @devpatil7 updates it.